app/views/layouts/_navbar.html.erb

```diff
@@ -20,18 +20,28 @@
             Meine Assets
           </a>
         </li>
+        <% if user_signed_in? %>
         <li class="nav-item dropdown">
           <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
             <i class="bi bi-collection-fill mobile-only"></i>
-            Meine Gruppen
+            <%= t(:my_groups) %>
           </a>
           <ul class="dropdown-menu bg-primary">
-            <li><a class="dropdown-item text-white" href="#">SWA-Lehrstuhl <i class="bi bi-patch-check-fill text-warning"></i></a></li>
-            <li><a class="dropdown-item text-white" href="#">Fachschaftsrat <i class="bi bi-patch-check-fill text-warning"></i></a></li>
-            <li><a class="dropdown-item text-white" href="#">Filmklub</a></li>
-            <li><a class="dropdown-item text-white" href="#">Spieleklub</a></li>
+            <li>
+              <%= link_to t(:all_groups), groups_path, class: "dropdown-item text-white" %>
+            </li>
+            <li><hr class="dropdown-divider"></li>
+            <% current_user.groups.each do |group| %>
+              <li>
+                <%= link_to group_path(group), class: "dropdown-item text-white" do %>
+                  <%= group.name %>
+                  <i class="bi bi-patch-check-fill text-warning"></i>
+                <% end %>
+              </li>
+            <% end %>
```

Comment:
Genauso wie in #116 führt dieser Abschnitt zu merge conflict; die Änderungen sind in beiden branches enthalten. Kann es sein, dass commits auf den falschen branch gepusht wurden?

Reply:
Ich habe auch gerade schon die Diskussion unter #116 gesehen, das Problem ist ja einfach, dass `ba/group-management-view` nicht frisch von `dev` gebranched wurde, sondern von `ba/mygroups-view`, nachdem dort schon gearbeitet wurde für den anderen Issue.
Aber für #7 muss die navbar nicht angepasst werden, sodass das kein Problem darstellen sollte.